packages/api/src/fixtures.ts

```diff
@@ -14,21 +21,76 @@ import {
 } from '@/utils/logParser';
 import { redisClient } from '@/utils/redis';
 
-import { SavedChartConfig, Tile } from './common/commonTypes';
-import { DisplayType } from './common/DisplayType';
-import * as config from './config';
-import { AlertInput } from './controllers/alerts';
-import { getTeam } from './controllers/team';
-import { findUserByEmail } from './controllers/user';
-import { mongooseConnection } from './models';
-import { AlertInterval, AlertSource, AlertThresholdType } from './models/alert';
-import Server from './server';
-
 const MOCK_USER = {
   email: 'fake@deploysentinel.com',
   password: 'TacoCat!2#4X',
 };
 
+const DEFAULT_LOGS_TABLE = 'default.otel_logs';
+const DEFAULT_TRACES_TABLE = 'default.otel_traces';
```

Comment:
Do we have a plan to move all these default variable in one file? So it can be easier to ref or setup with environment variables in future.

Reply:
I put these vars here since its only used for testing db instance